docs/distributions/arch/installation.md

```diff
@@ -7,7 +7,7 @@ You will need:
 
 - USB drive with at least 1GB
 - A way to plug it into your Mac (USB-C isn't USB-A)
-- A wired internet connection (i.e. USB-C to Enternet dongle) or wifi. If you need to install via wifi, you may use [this iso](https://github.com/Redecorating/archiso-mbp/releases), which has everything needed to follow the [wifi guide](https://wiki.t2linux.org/guides/wifi) and use wifi in the live environment.
+- A wired internet connection (i.e. USB-C to Enternet dongle) or wifi. If you need to install via wifi, you may use [this iso](https://github.com/NoaHimesaka1873/archiso-t2/releases), which has everything needed to follow the [wifi guide](https://wiki.t2linux.org/guides/wifi) and use wifi in the live environment.
```

Comment:
> which has everything needed to follow the [wifi guide](https://wiki.t2linux.org/guides/wifi) and use wifi in the live environment.

does your iso have the firmware automatically selected? If so then they won't need to follow the wifi guide to get wifi working in the live environment

Reply:
Yeah, my iso automatically selects firmware and also contains kernels and firmwares as packages for pacstrap. I think we need different guide to guide users to install those packages when pacstraping.